react/src/hooks/reactPaginationQueryOptions.tsx

```diff
@@ -0,0 +1,177 @@
+import { offset_to_cursor } from "../helper";
```

Comment:
I think it's better to remove this file from this PR because it cannot be used until backend.ai GraphQL supports the Relay pagination specification using a cursor.

Reply:
I agree. I'll remove this!